vaadin-date-time-picker-flow-testbench/src/main/java/com/vaadin/flow/component/datetimepicker/testbench/DateTimePickerElement.java

```diff
@@ -87,7 +150,102 @@ protected void setValue(String value) {
      * @return the value of the date time picker or an empty string if no date
      *         time is selected
      */
-    protected String getValue() {
-        return getPropertyString("value");
+    private String getValue() {
+        return getPropertyString(VALUE_PROPERTY);
+    }
+
+    /**
+     * Sets the selected date of the inner DatePicker as a string.
+     * <p>
+     * The value is based on ISO 8601 e.g. <code>YYYY-MM-DD</code>.
+     *
+     * @param value
+     *            the value to set
+     */
+    private void setDateValue(String value) {
+        $(DATE_PICKER_TAG_NAME).attribute(SLOT_ATTRIBUTE, DATE_PICKER_SLOT_NAME)
+                .first().setProperty(VALUE_PROPERTY, value);
```

Comment:
`$(DATE_PICKER_TAG_NAME).attribute(SLOT_ATTRIBUTE, DATE_PICKER_SLOT_NAME).first()`
^Would make the code more readable to extract these into private `getDatePicker()` and `getTimePicker()` methods.

Just an idea, not a blocker.

Reply:
Thanks, updated now. This was also a good chance to get rid of almost all of the String constants now that they are used only once or twice again (except for one constant that I left).